Jenkinsfile

```diff
@@ -112,6 +112,18 @@ def Tasks = [
         'set scalaJSLinkerConfig in helloworld ~= (_.withModuleKind(ModuleKind.CommonJSModule))' \
         helloworld/run \
         helloworld/clean &&
+    sbtretry ++$scala \
+        'set artifactPath in (helloworld, Compile, fastOptJS) := (crossTarget in helloworld).value / "helloworld-fastopt.mjs"' \
```

Comment:
Because that would tie the behavior of the sbt plugin to something that's 1) Node.js-specific and 2) even still experimental in Node.js. As long as it's experimental and/or not widespread, I'd rather users do that themselves.

Reply:
Depending on the outcome of the other comment (rename if necessary) this might not be necessary at all anymore. I prefer this, as it would hide NodeJS details inside NodeJS.